### DISCUSSION TITLE
Two-Factor Authentication Issue

### DISCUSSION THREAD

**Discussion by tptkds** [supabase/supabase#27802] [Questions]
## Issue with Two-Factor Authentication

I am experiencing an issue with logging into my Supabase account. I have been logged out and prompted to enter a two-factor authentication code. However, I did not set up two-factor authentication on my account.

### URL
`https://supabase.com/dashboard/sign-in-mfa?category=dashboard_bug&error=server_error&error_description=sql%3A+Scan+error+on+column+index+14%2C+name+%22encrypted_password%22%3A+converting+NULL+to+string+is+unsupported&message=Error%3A+This+endpoint+requires+a+Bearer+token&returnTo=%2Fsupport%2Fnew&subject=Failed+to+retrieve+factors#error=server_error&error_description=sql%253A+Scan+error+on+column+index+14%252C+name+%2522encrypted_password%2522%253A+converting+NULL+to+string+is+unsupported`

### Error Message
**Additionally, I am seeing the following error message:** 
Failed to retrieve factors
Error: This endpoint requires a Bearer token

Try refreshing your browser, but if the issue persists, please reach out to us via support.

I have tried refreshing my browser, but the issue persists. When I click the "Contact support" button, it redirects me back to the login page instead of providing support options.

### Login Redirect URL
`https://supabase.com/dashboard/sign-in?category=dashboard_bug&subject=Failed+to+retrieve+factors&message=Error%3A+This+endpoint+requires+a+Bearer+token&returnTo=%2Fsupport%2Fnew`

### Screenshot
![image](https://github.com/supabase/supabase/assets/58039782/817e5f44-d471-4af1-a00f-9aac620f9bbf)

**Comment by AdrianPodraza** [comment]
Did you somehow resolve this issue?

**Reply by tptkds** [reply]
> Did you somehow resolve this issue?

I couldn't resolve it. As an alternative, I created a new local account. 
Additionally, the account with the issue was using GitHub login.

**Comment by GaryAustin1** [comment]
Seems like you have to contact them.   During an outage yesterday on new accounts things were left in a bad state as I read this:
https://status.supabase.com/#past-incidents

**Reply by tptkds** [reply]
> Seems like you have to contact them. During an outage yesterday on new accounts things were left in a bad state as I read this:
> https://status.supabase.com/#past-incidents

Thanks for the comment. However, when I click the "Contact us" button, it redirects me to the login page, so I couldn't reach out directly. I have created an issue, so please refer to it: https://github.com/supabase/supabase/issues/27812

**Reply by GaryAustin1** [reply]
support@supabase.io is their email.   I can't do anything for you.

**Reply by tptkds** [reply]
> [support@supabase.io](mailto:support@supabase.io) is their email. I can't do anything for you.

Thank you for sharing the email.

**Comment by helloromero** [comment]
for anyone having this issue the only fix I found was to use a completely different browser.  I was experiencing it in Brave (Version 1.69.168 Chromium: 128.0.6613.138 (Official Build) (arm64)) and even after cleaning cookies, sessions cache etc I had no luck.

But going thought the login process again via Safari worked as normal.
